php/meterpreter/meterpreter.php

```diff
@@ -42,8 +42,27 @@ function register_command($c, $i) {
   }
 }
 
+# Debugging payload definitions
+define("DEBUGGING", false);
+define("DEBUG_LOGFILE_PATH", "");
+
+function my_logfile($str) {
+  if (DEBUGGING AND DEBUG_LOGFILE_PATH != "") {
```

Comment:
Is `and` required here? 👀 

https://stackoverflow.com/questions/2803321/and-vs-as-operator

```suggestion
  if (DEBUGGING && DEBUG_LOGFILE_PATH != "") {
```